temporalio/client.py

```diff
@@ -711,6 +722,7 @@ async def execute_workflow(
         rpc_metadata: Mapping[str, str] = {},
         rpc_timeout: Optional[timedelta] = None,
         request_eager_start: bool = False,
+        priority: Optional[temporalio.common.Priority] = None,
```

Comment:
`execute_workflow` isn't passing the new parameter on to `start_workflow`.

Reply:
Thank you